AppDir/AppRun

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+HERE="$(dirname "$(readlink -f "${0}")")"
+export PATH="${HERE}"/usr/bin/:"${PATH}"
+EXEC=$(grep -e '^Exec=.*' "${HERE}"/*.desktop | head -n 1 | cut -d "=" -f 2 | cut -d " " -f 1)
+LD_LIBRARY_PATH=$HERE exec "${HERE}/${EXEC}" $@
```

Comment:
is `LD_LIBRARY_PATH` necessary? I think we compile osu as a self-contained app.

Reply:
We do, however osu uses external libs like BASS... which is located at the same directory as the osu bin. Running osu! without LD_LIBRARY_PATH set will crash osu.

Reply:
See https://github.com/ppy/osu/blob/master/README.md for more info.